File: 7-kyu/simple-fun-number-152-invite-more-women/simple-fun-number-152-invite-more-women.java
public class Kata {
    public static boolean inviteMoreWomen(int[] l) {
        int soma = 0;
  for(int i = 0; i < l.length; i++){
    soma += l[i];
  }
  if(soma > 0) return true;
  else return false;
    }
}

File: 7-kyu/simple-fun-number-152-invite-more-women/simple-fun-number-152-invite-more-women.c
#include <stddef.h>
#include <stdbool.h>
​
bool invite_more_women(const int array[/* count */], size_t count)
{
  int soma = 0;
  for(size_t i = 0; i < count; i++){
    soma += array[i];
  }
  if(soma > 0) return true;
  else return false;
}